partner_contact_job_position/README.rst

```diff
@@ -28,23 +28,24 @@ To use this module, you need to:
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/134/10.0
+   :target: https://runbot.odoo-community.org/runbot/134/11.0
```

Comment:
No new README by fragments?

Reply:
I was waiting your super helpful comment of the other PR

Reply:
:+1: